tests/Operation/WatchFunctionalTest.php

```diff
@@ -484,16 +579,8 @@ public function testTypeMapOption(array $typeMap, $expectedChangeDocument)
 
         $changeStream->next();
         $this->assertTrue($changeStream->valid());
-        $changeDocument = $changeStream->current();
-
-        // Unset the resume token and namespace, which are intentionally omitted
```

Comment:
Thanks to `assertMatchesDocument()`, we can ignore extra fields in the actual document that are not present in our expected value. This allows us to do without these manual unsets.

Reply:
Does that also address the extra "clusterTime" information that made tests fail?